components/viewer/ViewProjectObj.vue

```diff
@@ -39,6 +39,7 @@
           </template>
           <ViewScores :scores="obj.scores" />
           <ViewRequirements :requireds="obj.requireds" />
+          <br />
```

Comment:
🚫 Spacing is done with CSS!

Reply:
The simplest way would be to add a `margin-top: 1rem;` to the `.obj-text` style to add spacing between the object text and the object stack above.